packages/nuxt/module.cjs

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const kit = require('@nuxt/kit');
+const vueDemi = require('vue-demi');
+
+const module$1 = kit.defineNuxtModule({
+  name: "pinia",
+  configKey: "pinia",
+  defaults: {
+    disableVuex: true
```

Comment:
I think this file was included by accident

Reply:
Ah I mistakenly replaced it on the last commit.